apps/coordinator/src/reducers/transactionReducer.js

```diff
@@ -281,12 +282,16 @@ function updateOutputAmount(state, action) {
 
 function finalizeOutputs(state, action) {
   let unsignedTransaction;
+  const rbfSequence = state.rbfEnabled ? 0xfffffffd : 0xffffffff;
   // First try to build the transaction via PSBT, if that fails (e.g. an input doesn't know about its braid),
   // then try to build it using the old TransactionBuilder plumbing.
   try {
     const args = {
       network: state.network,
-      inputs: state.inputs.map(convertLegacyInput),
+      inputs: state.inputs.map((input) => ({
+        ...convertLegacyInput(input),
+        sequence: rbfSequence,
```

Comment:
So I can see an [implementation](https://github.com/caravan-bitcoin/caravan/pull/114/files#diff-db5fc022ab08b92fb2a606342674432f918748c762c9fcb907bc43d3531d512bR133) for the sequence field has been added in the fees package which works with PSBT V2. But as this PR stands, I think the sequence is not persisting in the inputs
<img width="2056" alt="image" src="https://github.com/user-attachments/assets/f2a4bd00-a1f8-4cf9-b959-77ee8d4bc13d">
`inputData` logging is coming from `packages/caravan-psbt/src/psbtv0/psbt.ts:L90`

Working with testnet v4 (merged in #131)

Reply:
Thanks for the feedback .... I shall move on to resolving this issue and completing the fees package integration, once I am done with completing the last round of reviews on the fees package and getting it merged \:)